packages/trader/src/AppV2/Components/TradeParameters/RiskManagement/__tests__/take-profit-and-stop-loss-input.spec.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mockStore } from '@deriv/stores';
+import ModulesProvider from 'Stores/Providers/modules-providers';
+import * as utils from 'AppV2/Utils/trade-params-utils';
+import { TTradeStore } from 'Types';
+import TraderProviders from '../../../../../trader-providers';
+import TakeProfitAndStopLossInput from '../take-profit-and-stop-loss-input';
+
+type TResponse = {
+    proposal: Record<string, unknown>;
+    echo_req: { contract_type: string };
+    subscription: Record<string, unknown>;
+    error?: Record<string, never> | { details: { field: string }; message: string };
+};
+
+const data_testid = 'dt_input_with_steppers';
+const acc_content = 'Note: Cannot be adjusted for ongoing accumulator contracts.';
```

Comment:
```suggestion
const accu_content = 'Note: Cannot be adjusted for ongoing accumulator contracts.';
```
for a moment, i thought acc = account ))